cmd/bnscli/clitests/batch_proposal.test

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+# Combine all messages in a single file to build a batch transaction containing
+# all of them.
+# The same result could be achieved by running bellow commands in a subprocess
+# and stream the combined output into "as-batch" command.
+msgstream=`tempfile`
+bnscli send-tokens -src 'seq:foo/src/1' -dst "seq:foo/dst/1" -amount "2 IOV" -memo "sending 2 IOV" >> $msgstream
+bnscli send-tokens -src 'seq:foo/src/2' -dst "seq:foo/dst/2" -amount "9 DOGE" -memo "sending 9 DOGE" >> $msgstream
+bnscli send-tokens -src 'seq:foo/src/3' -dst "seq:foo/dst/3" -amount "7 BTC" -memo "sending 7 BTC" >> $msgstream
+bnscli release-escrow -escrow 04 >> $msgstream
+
+bnscli as-batch < $msgstream \
+	| bnscli as-proposal -start "2021-01-01 11:11" -electionrule 3 -title "my proposal" -description "yet another proposal" \
```

Comment:
Interesting here you parse an int into the 8 byte hex, where escrow assumes those 8 bytes...

Reply:
The 8 bytes is more general, but the integer works great with stuff defined in the genesis file (and I assume in reality most of these commands will be used with genesis-defined objects)

Reply:
My motivation was exactly the declaration in the genesis. I agree that using hex encoded value is more powerful. The current implementation might be confusing (electionrule is an int, but escrow is a hex). Do you think I should change it to a hex value? If yes, than [validation discuss](https://github.com/iov-one/weave/pull/735#discussion_r291490219) is relevant.

Reply:
I would actually change the escrow one, just notices you did it (in my eyes) better here.

Reply:
I can do this, it's an easy change in both cases.

Aren't you afraid that using a number explicitly will make it harder later to provide the value? For example, when I create an escrow, I will get back a byte array. I can decode it to a number if I know it represents a number. Otherwise, I got bytes and I don't know how to convert it.

Reply:
Yes, but if 15 is valid, it could either be one byte hex, or an int.

I would assert either:
* integer, decimal encoded
* 8 bytes, hex encoded

and then it is impossible that you intend an integer and it is interpreted as hex bytes.

I can go either way which is more correct to enter, but we should enforce the format when parsing